srml/staking/src/lib.rs

```diff
@@ -325,6 +325,17 @@ pub struct EraRewards {
 	rewards: Vec<u32>,
 }
 
+impl EraRewards {
+	/// add the reward to the validator at index given. Index must valid.
```

Comment:
```suggestion
	/// add the reward to the validator at the given valid `index`.
	///
	/// Invalid index will be ignored.
```

Reply:
hmm yes and no you give an index that is beyond the size of elected vec then its points will be discarded but not distributed to other thus it is not a no-op. at some point making this a no-op can be done in new_era.

Reply:
I see, then no need for `/// Invalid index will be ignored.`